### ISSUE TITLE
Bracket coloring disappears when I change settings

### ISSUE THREAD

**Issue by davewill** [CoenraadS/Bracket-Pair-Colorizer-2#275]
I'm using the extension with C++, The problem also manifests in the JSON settings files. Each time I change a setting in Bracket Pair Colorizer 2, I lose bracket coloring until I restart VSC.

Before:
![Annotation 2020-05-20 121251](https://user-images.githubusercontent.com/730907/82488107-36982780-9a94-11ea-924b-ba2818671fb1.png)

Change "Show Brackets in Gutter"

Then:
![Annotation 2020-05-20 121551](https://user-images.githubusercontent.com/730907/82488130-43b51680-9a94-11ea-9818-4021d1d9099b.png)

All brackets have turned white, and I have to restart VSC to get the colors back. Other features like brackets in the gutter, highlighting the current scope, and BracketSelection seem to still work. It makes experimenting with colors or other settings tedious beyond comprehension. 

I tried disabling all other extensions, but get the same behavior.

User setting.json:
{
    "terminal.integrated.shell.linux": "/bin/bash",
    "git.suggestSmartCommit": false,
    "arduino.enableUSBDetection": false,
    "editor.matchBrackets": "never",
    "bracket-pair-colorizer-2.highlightActiveScope": true,
    "typescript.format.placeOpenBraceOnNewLineForFunctions": true,
    "bracket-pair-colorizer-2.showBracketsInGutter": true,
}

Extensions:
Name: Arduino
Id: vsciot-vscode.vscode-arduino
Description: Arduino for Visual Studio Code
Version: 0.3.0
Publisher: Microsoft
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=vsciot-vscode.vscode-arduino

Name: Bracket Pair Colorizer 2
Id: coenraads.bracket-pair-colorizer-2
Description: A customizable extension for colorizing matching brackets
Version: 0.1.3
Publisher: CoenraadS
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=CoenraadS.bracket-pair-colorizer-2

Name: C/C++
Id: ms-vscode.cpptools
Description: C/C++ IntelliSense, debugging, and code browsing.
Version: 0.28.0
Publisher: Microsoft
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools

Name: Cortex-Debug
Id: marus25.cortex-debug
Description: ARM Cortex-M GDB Debugger support for VSCode
Version: 0.3.6
Publisher: marus25
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=marus25.cortex-debug

Name: Remote - WSL
Id: ms-vscode-remote.remote-wsl
Description: Open any folder in the Windows Subsystem for Linux (WSL) and take advantage of Visual Studio Code's full feature set.
Version: 0.44.2
Publisher: Microsoft
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl

VSCode version info:
Version: 1.45.1 (user setup)
Commit: 5763d909d5f12fe19f215cbfdd29a91c0fa9208a
Date: 2020-05-14T08:27:35.169Z
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18362

**Comment by wapenshaw** [comment]
Yep, Happens when I change any setting. Have to restart VSCode completely. 
![image](https://user-images.githubusercontent.com/739546/83971184-7dc24d00-a8c9-11ea-866f-fddf0fb58d25.png)

**Comment by weppami** [comment]
@wapenshaw  same here

**Comment by wapenshaw** [comment]
> @wapenshaw  same here

Hey, its still the same but instead of restarting, I just use the "Developer: reload window" option in the command palette.

Just to be clear on what I was reporting about. If I add/remove/change any entry in settings.json, the bracket colors disappear.

**Comment by pdxwolfy** [comment]
Same here with bracket colors disappearing. The "Developer: reload window" trick doesn't work for me.

**Comment by Jerome1337** [comment]
Sometimes, even after a restart this is still not working.

Some options seems breaking it instead configuring it 😄
